CMakeLists.txt

```diff
@@ -1,56 +1,34 @@
 cmake_minimum_required(VERSION 3.19)
-project(xania)
-
-####
-# Shell out to the Makefile to get all the paths it installed for us
-set(CMAKE_MAKE_CONFIG_FILE ${CMAKE_CURRENT_BINARY_DIR}/config-from-make.cmake)
```

Comment:
I checked out the `pr/290` branch on my box and did a distclean, then `make install test` -- broken in a couple of ways currently. First, seems to report using gcc/g++-9, even though I have 10 and 11 installed:

```
-- The C compiler identification is GNU 9.3.0
-- The CXX compiler identification is GNU 9.3.0

```

Second, it barfs on conan:
```
-- Conan: checking conan executable
-- Conan: Found program /home/rsnell/.local/bin/conan
CMake Error at conan.cmake:807 (message):
  Conan --version failed='1'
Call Stack (most recent call first):
  CMakeLists.txt:13 (conan_check)

```

Reply:
OK I figured out why I was getting that conan.cmake error -- I had an ancient version of the `conan` wrapper script left over in my `~/.local/bin/` directory, I think from when I had attempted to install conan directly myself using `pip3` last year.  I've removed this and that error goes away, and it attempts to do the conan download instead.  However that is failing with permission denied like this:

```
'chmod' '+x' '/home/rsnell/dev/xania/cmake-build-debug/conda-4.10.3-installer.sh'
'/home/rsnell/dev/xania/cmake-build-debug/conda-4.10.3-installer.sh' '-u' '-b' '-p' '/home/rsnell/dev/xania/cmake-build-debug/.tools//conda-4.10.3'
CMake Error at CMakeLists.txt:36 (execute_process):
  execute_process error getting child return code: Permission denied

```

Reply:
Looks like it's failing in the CI for the same reason.

Reply:
yes, I'm currently trying to fix that, but I'm testing with the network from my phone and the conda dowloads is difficult to test because of the (very) limited bandwidth... I think I'll need to wait I'm back home for testing correctly the network parts...

Reply:
No worries, there is no urgency :)

Reply:
should be fixed now